wait-for-it.sh

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env bash
```

Comment:
When you are including 3rd party files which you didn't write in the repository, you should add a reference to the original author and file license to the header.

Something along the lines of "# Originally from https://github.com/vishnubob/wait-for-it/blob/master/wait-for-it.sh (MIT license)".

Reply:
Yes, I had forgotten about it and will be updating the author reference shortly.